datasimulator/main.py

```diff
@@ -144,9 +143,13 @@ def run_submission_order_generation(graph, data_path, node_name=None):
     else:
         submission_order = graph.generate_submission_order()
 
-    with open(os.path.join(data_path, "DataImportOrderPath.txt"), "w") as outfile:
-        for node in submission_order:
-            outfile.write(node.name + "\t" + node.category + "\n")
+    file_path = os.path.join(data_path, "DataImportOrderPath.txt")
+    path_exists = os.path.exists(data_path)
+    if not path_exists:
+        logger.error("Cannot create file because path does not exist. Did you create a 'test-data' folder?")
```

Comment:
Is data_path always going to be `test-data`? If not then this should actually just be changed to reflect whatever `data_path` is.

Reply:
there is no default, it's set [here](https://github.com/uc-cdis/data-simulator/blob/b02b05e576f690d6907670fdb42d0c187518ceb2/datasimulator/main.py#L25)

Reply:
I'll change to just point the the variable